modules/RTC/ScreenObtainer.js

```diff
@@ -216,33 +216,30 @@ const ScreenObtainer = {
             getDisplayMedia = navigator.mediaDevices.getDisplayMedia.bind(navigator.mediaDevices);
         }
 
-        const { desktopSharingFrameRate } = this.options;
-        const setScreenSharingResolutionConstraints = browser.isChromiumBased()
-            && this.options?.testing?.setScreenSharingResolutionConstraints;
+        const audio = this._getAudioConstraints();
         let video = {};
-
-        // Allow users to seamlessly switch which tab they are sharing without having to select the tab again.
-        if (browser.isChromiumBased() && browser.isVersionGreaterThan(106)) {
-            video.surfaceSwitching = 'include';
-        }
+        const { desktopSharingFrameRate } = this.options;
 
         if (typeof desktopSharingFrameRate === 'object') {
             video.frameRate = desktopSharingFrameRate;
         }
 
-        // Capturing the screenshare at very high resolutions restricts the framerate. Therefore, skip this hack when
-        // the capture framerate is > 5 fps.
-        if (setScreenSharingResolutionConstraints && !(desktopSharingFrameRate?.max > SS_DEFAULT_FRAME_RATE)) {
-            // Set bogus resolution constraints to work around
-            // https://bugs.chromium.org/p/chromium/issues/detail?id=1056311
-            video.height = 99999;
-            video.width = 99999;
-        }
+        // At the time of this writing 'min' constraint for fps is not supported by getDisplayMedia on any of the
+        // browsers. getDisplayMedia will fail with an error "invalid constraints" in this case.
+        video.frameRate && delete video.frameRate.min;
 
-        const audio = this._getAudioConstraints();
+        if (browser.isChromiumBased()) {
+            // Allow users to seamlessly switch which tab they are sharing without having to select the tab again.
+            browser.isVersionGreaterThan(106) && (video.surfaceSwitching = 'include');
 
-        // At the time of this writing 'min' constraint for fps is not supported by getDisplayMedia.
-        video.frameRate && delete video.frameRate.min;
+            // Set bogus resolution constraints to work around
+            // https://bugs.chromium.org/p/chromium/issues/detail?id=1056311 for low fps screenshare. Capturing SS at
+            // very high resolutions restricts the framerate. Therefore, skip this hack when capture fps > 5 fps.
+            if (desktopSharingFrameRate?.max <= SS_DEFAULT_FRAME_RATE) {
```

Comment:
It seems before we were applying this for all browsers but now only for chromium based. This is intentional and it is chrome specific?

Reply:
It is Chrome specific and before also we had the Chrome check - https://github.com/jitsi/lib-jitsi-meet/pull/2195/files#diff-ce2d682b5181dd9a7081e3a971c936cf99e8c6d24644c7dafcd0ee10e85d30f5L220